user.go

```diff
@@ -5,15 +5,33 @@ import "gopkg.in/mgo.v2/bson"
 type User struct {
 	Id       bson.ObjectId `json:"id" bson:"_id,omitempty"`
 	Username string        `json:"username" bson:"username"`
+	FullName string 	`json:"fullname" bson:"fullname"`
+	Email	 string `json:"email" bson:"email"`
+	PasswordHash string `json:"password" bson:"password"`
```

Comment:
It seems odd to me that the internal field is called `PasswordHash` but the external ones only `password`. Did you have any reason for that?